index.html

```diff
@@ -7216,8 +7216,8 @@ <h2>application/vc</h2>
         </p>
 
         <p>
-This media type can be used for credentials secured using an
-[=enveloping proof=].
+This media type can be used within [=enveloping proof=] when denoting the
```

Comment:
```suggestion
This media type can be used within [=enveloping proofs=] when denoting the
```

Reply:
Given `contents of the [singular] envelope` that follows, I think keeping the `proof` singular makes sense. Adding `an` fixes the grammar.
```suggestion
This media type can be used within an [=enveloping proof=] when denoting the
```